### PR TITLE
Add documentation for two shared CI tasks

### PR DESCRIPTION
**Pull request by aramprice**

### CODE REVIEW

**Review by ramonskie — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -65,3 +65,26 @@ export OS="windows2019"
...
 export VM_EXTENSIONS="[50GB_ephemeral_disk]"
 ```
+
+## Shared Concourse tasks
+
+This repository provides a couple helpful Concourse tasks in `ci/tasks/shared` that can help keep the Golang package vendored in your BOSH release up to date, and bump dependencies.
+
+### ci/tasks/shared/bump-golang-package
+
+The `bump-golang-package` task runs `bosh vendor-package` to automatically update the version of Ruby vendored into your own BOSH release.
```
Comment (selzoc) [inline README.md]: Probably meant `go` here instead of `Ruby`?

Comment (aramprice) [reply README.md]: Fixed, thanks!

**Review by selzoc — APPROVED** [review]

**Comment by jpalermo** [comment]
Thanks!

**Pull request merged**